tox.ini

```diff
@@ -91,16 +91,16 @@ commands = pip wheel -w {envtmpdir}/build --no-deps .
 [testenv:embed]
 description = embed dependencies into virtualenv.py
 skip_install = true
-changedir = {toxinidir}/bin
+changedir = {toxinidir}/tasks
 extras =
-commands = python rebuild-script.py
+commands = python update_embedded.py
 
 [testenv:upgrade]
 description = upgrade pip/wheels/setuptools to latest
 skip_install = true
 extras = pip >= 18.1.0
```

Comment:
@gaborbernat this is what I was talking about on Twitter

Reply:
that's indeed a bug 😅

Reply:
PR incoming

Reply:
should be deps instead

Reply:
Sure :)